modules/caddyhttp/matchers.go

```diff
@@ -131,7 +131,10 @@ type (
 		// to spoof request headers. Default: false
 		Forwarded bool `json:"forwarded,omitempty"`
 
+		// cidrs and zones vars should aligned always in the same 
```

Comment:
Hmm, looks like gofmt isn't happy with this. Please run `gofmt -s -w modules/caddyhttp/matchers.go` to fix it up